### ISSUE TITLE
Release?

### ISSUE THREAD

**Issue by krische** [jenkinsci/groovy-events-listener-plugin#52]
Can we get a release? It looks like there hasn't been a release since #50 was merged. This is necessary to run the plugin in newer versions of Jenkins.

**Comment by markyjackson-taulia** [comment]
Yup, I will have it out by Sunday

**Comment by Irialad** [comment]
I just cloned this repo, modified `build.gradle` to set
```Groovy
jenkinsPlugin {
    coreVersion = '2.102'
```
and then ran `./gradlew server` to get this:

```
org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin$DescriptorImpl in file:/home/rrodrigues/code/ext/groovy-events-listener-plugin/build/classes/main/ might be dangerous, so rejecting; see https://jenkins.io/redirect/class-filter/
Failed to load /home/rrodrigues/code/ext/groovy-events-listener-plugin/work/org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin.xml
java.io.IOException: Unable to read /home/rrodrigues/code/ext/groovy-events-listener-plugin/work/org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin.xml
        at hudson.XmlFile.unmarshal(XmlFile.java:182)
        at hudson.XmlFile.unmarshal(XmlFile.java:162)
        at hudson.model.Descriptor.load(Descriptor.java:893)
        at hudson.model.Descriptor$load.callCurrent(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:52)
</snip>
```
So unless you're addressing that somewhere other than master, I'm thinking a new release is going to have the issue.

**Comment by markyjackson-taulia** [comment]
Well yeah, that is what happens prior to a release, you test. If test has a sad path you debug.
That is all part of my process for releasing. So what is on master now may not be what goes in the final release

**Comment by Irialad** [comment]
Then, since you're already looking at this, I'll not open what would be a duplicative issue.
Cheers!

**Comment by markyjackson-taulia** [comment]
Thank you kindly. Truth be told, I do not think it is fair of me to keep my process a secret and I intend to change that and document more what the roadmap is and the process, I think that is fair and transparent.

**Comment by markyjackson-taulia** [comment]
@Irialad so I owe you and others that Rob and I will get that, I promise

**Comment by darxriggs** [comment]
I will cover at least the below warning (also mentioned by @Irialad) in an upcoming pull request.

```org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin$DescriptorImpl in file:/home/rrodrigues/code/ext/groovy-events-listener-plugin/build/classes/main/ might be dangerous, so rejecting; see https://jenkins.io/redirect/class-filter/```

**Comment by markyjackson-taulia** [comment]
Pending #53

**Comment by darxriggs** [comment]
I also want to create another pull request to be contained in the next release after #53 is merged.

**Comment by markyjackson-taulia** [comment]
I just cut the release major release but we can do an incremental: https://github.com/jenkinsci/groovy-events-listener-plugin/releases/tag/1.017

Let's open a new issue to track new pr you want to include

**Comment by darxriggs** [comment]
@markyjackson-taulia I am just using this issue for this comment in the lack of another possibility to comment on a release.

Here is my request:

Can you update the release text of https://github.com/jenkinsci/groovy-events-listener-plugin/releases/tag/1.016 to only contain the release version `1.016` instead of the full text `1.016: Merge pull request #44 from darxriggs/documentation-improvements`?

This would make it consistent with the other releases.

**Comment by markyjackson-taulia** [comment]
Yes, totally! I want to reshape our changelog to follow the new standard of the overall Jenkins community

**Comment by Irialad** [comment]
Just a heads up, https://plugins.jenkins.io/groovy-events-listener-plugin
says that 1.017 was released 5 months ago. I'm guessing this indicates that this release of 1.017 did not actually update the previously released 1.017?

**Comment by markyjackson-taulia** [comment]
That is correct @Irialad 
There is a transitioning happening with plugins and the wiki and how that all gets tied together

**Comment by darxriggs** [comment]
@markyjackson-taulia that's confusing when a release on _plugins.jenkins.io_ differs from the tag on _github.com_. Please also have a look at [gradle.properties](https://github.com/jenkinsci/groovy-events-listener-plugin/blob/1.017/gradle.properties#L4) and keep it in mind for the next release. It still contains `version = 1.015-SNAPSHOT`.
